cluster/gce/gci/configure-helper.sh

```diff
@@ -1461,40 +1462,97 @@ function docker-installed {
     fi
 }
 
+# util function to add a docker option to daemon.json file only if the daemon.json file is present.
+# accepts only one argument (docker options)
+function addockeropt {
+	DOCKER_OPTS_FILE=/etc/docker/daemon.json
+	if [ "$#" -lt 1 ]; then
+	echo "No arguments are passed while adding docker options. Expect one argument"
+	exit 1
+	elif [ "$#" -gt 1 ]; then
+	echo "Only one argument is accepted"
+	exit 1
+	fi
+	# appends the given input to the docker opts file i.e. /etc/docker/daemon.json file
+	if [ -f "$DOCKER_OPTS_FILE" ]; then
+	cat >> "${DOCKER_OPTS_FILE}" <<EOF
+  $1
+EOF
+	fi
+}
+
+function set_docker_options_non_ubuntu() {
+  # set docker options mtu and storage driver for non-ubuntu
+  # as it is default for ubuntu
+  if ! command -v apt-get >/dev/null 2>&1; then
+    echo "Unable to add docker options. Bailing out..."
```

Comment:
maybe make this message more descriptive - tell it's an Ubuntu and which options will not be added

Reply:
We only have COS and Ubuntu, right

can we do: 
set_docker_options_for_cos instead